src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -174,6 +182,12 @@ public StreamingResponseBody streamEvents(
             final AtomicBoolean connectionReady = closedConnectionsCrutch.listenForConnectionClose(request);
             Counter consumerCounter = null;
             EventStream eventStream = null;
+            EventConsumer eventConsumer = null;
+            final String kafkaClientMetricsName = MetricRegistry.name(
+                    "lola",
+                    client.getClientId(),
+                    eventTypeName,
```

Comment:
Do you consider using MetricUtils to generate the name? It replaces `.` in event type name with `#`.